querydsl-core/src/test/java/com/querydsl/core/types/ExpressionUtilsTest.java

```diff
@@ -148,6 +150,12 @@ public void In() {
     }
 
     @Test
+    public void In_SubQuery() {
+        String s = ExpressionUtils.in(str, new SubQueryExpressionImpl<String>(String.class, new DefaultQueryMetadata())).toString();
+        assertTrue(s.startsWith("str in com.querydsl.core.DefaultQueryMetadata@c"));
```

Comment:
Alternatively we can use some SQL-like syntax to serialize module independent subquery expressions.

Now it just calls just toString() on the metadata.